app_data/vocabularies/resource_types.yaml

```diff
@@ -0,0 +1,1057 @@
+- id: publication
+  icon: file alternate
+  props:
+    csl: report
+    datacite_general: Text
```

Comment:
shall we use [Report](https://datacite-metadata-schema.readthedocs.io/en/4.6/appendices/appendix-1/resourceTypeGeneral/#report) instead?
See below